pkg/network/BUILD.bazel

```diff
@@ -1,8 +1,55 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
-    srcs = ["reserved_macs.go"],
+    srcs = [
+        "network.go",
+        "podinterface.go",
+        "podnic.go",
+    ],
     importpath = "kubevirt.io/kubevirt/pkg/network",
     visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/network/cache:go_default_library",
+        "//pkg/network/dhcp:go_default_library",
+        "//pkg/network/driver:go_default_library",
+        "//pkg/network/errors:go_default_library",
+        "//pkg/network/infraconfigurators:go_default_library",
+        "//pkg/network/link:go_default_library",
+        "//pkg/virt-launcher/virtwrap/api:go_default_library",
```

Comment:
We should not depend inwards on virt-* packages.

Reply:
well is where the libvirt Domain structs are living, we cannot remove this dep, maybe problem is that the api.Domain and friends cannot live there, since it's also used by virt-handler.